src/Fluid/Flu_FixUp_Restrict.cpp

```diff
@@ -361,6 +361,24 @@ void Flu_FixUp_Restrict( const int FaLv, const int SonFluSg, const int FaFluSg,
                               Passive,
                               CheckMinTemp_No, NULL_REAL, Emag, EoS_DensEint2Temp_CPUPtr,
                               EoS_AuxArray_Flt, EoS_AuxArray_Int, h_EoS_Table );
+
+//       check output temperature initial guess
+#        ifdef GAMER_DEBUG
+         const real Temp_IG = amr->patch[FaFluSg][FaLv][FaPID]->fluid[TEMP_IG][k][j][i];
+         if (  Hydro_CheckUnphysical( UNPHY_MODE_SING, &Temp_IG, "output temperature initial guess", ERROR_INFO, UNPHY_VERBOSE )  )
```

Comment:
Add one empty line above for readability.

Reply:
I added a line.